root/main/index.js

```diff
@@ -167,34 +170,60 @@ const Homepage = ({
 
     <div className="feature-column" style={{clear: 'both', paddingTop: '1%'}}>
       <h2>{l('Recent Additions')}</h2>
+      {withSafeBrowsing ? (
+        <div className="banner">
+          {l('Cover art on the homepage is disabled for Chrome-based browsers which are known to issue incorrect phishing warning since the 11th of November 2018, see {ticket_link|ticket CAA-116} for follow-up.<br/>Phishing warning may show up on any other page with cover art. You are encouraged to {report_link|report errors to Google Safe Browsing}. Sorry for the inconvenience!', {
```

Comment:
A few small grammar tweaks:
```suggestion
          {l('Cover art on the homepage is disabled for Chrome-based browsers, which are known to issue an incorrect phishing warning since the 11th of November 2018; see {ticket_link|ticket CAA-116} for follow-up.<br/>A phishing warning may show up on any other page with cover art. You are encouraged to {report_link|report errors to Google Safe Browsing}. Sorry for the inconvenience!', {
```